docs/concepts/projects/dependencies.md

```diff
@@ -636,21 +664,13 @@ lint = [
 Once groups are defined, the `--all-groups`, `--no-default-groups`, `--group`, `--only-group`, and
 `--no-group` options can be used to include or exclude their dependencies.
 
-!!! tip
-
-    The `--dev`, `--only-dev`, and `--no-dev` flags are equivalent to `--group dev`,
-    `--only-group dev`, and `--no-group dev` respectively.
-
-uv requires that all dependency groups are compatible with each other and resolves all groups
-together when creating the lockfile.
-
-If dependencies declared in one group are not compatible with those in another group, uv will fail
-to resolve the requirements of the project with an error.
+The `--dev`, `--only-dev`, and `--no-dev` flags are equivalent to `--group dev`, `--only-group dev`,
+and `--no-group dev` respectively.
 
 !!! note
 
-    If you have dependency groups that conflict with one another, resolution will fail
-    unless you explicitly [declare them as conflicting](./config.md#conflicting-dependencies).
+    `include-group` is not supported from the CLI ([#9054](https://github.com/astral-sh/uv/issues/9054)).
+    Those edits must be made manually.
```

Comment:
This seems like a distracting/strange thing to include in the documentation.

Reply:
Since there was an issue open about it, I thought it might save someone from creating another issue.  :)

I moved the notice to a comment in the example but wouldn't mind at all if your preference is to remove that too.